### PR TITLE
feat: include financial contributor contact email in notification

### PR DESCRIPTION
**Pull request by snitin315**
Resolve https://github.com/opencollective/opencollective/issues/2297

#### Short description of what this resolves:
Now new backer notification will display contact email of a financial contributor if available.

#### Changes proposed in this pull request:

If email is available display it in about section.

# Screenshot

notification

### CODE REVIEW

**Comment by snitin315** [comment]
@Betree any update on this one?

File: server/models/Collective.js [hunk 1]
```diff
@@ -1472,12 +1472,14 @@ export default function (Sequelize, DataTypes) {
 
     const urlPath = await this.getUrlPath();
     const memberCollective = await models.Collective.findByPk(member.MemberCollectiveId, sequelizeParams);
+    const memberCollectiveUser = await models.User.findByPk(member.CreatedByUserId, sequelizeParams);
```
Comment (znarf) [inline server/models/Collective.js:1475]: `CreatedByUserId` fields should not be used for such thing, we should query `User` with the `CollectiveId` instead.

**Comment by Betree** [comment]
As per the latest discussions in https://github.com/opencollective/opencollective/issues/2297, we should only include the email if collective.type is `USER`.

File: server/models/Collective.js (hunk 1 above)
Comment (snitin315) [reply server/models/Collective.js:1476]: like this `await models.User.findByPk(member.CollectiveId, sequelizeParams)` ?

Comment (Betree) [reply server/models/Collective.js:1476]: `member.CollectiveId` is a collective ID (not a user ID). You can try something like this instead:

```es6
let memberCollectiveUser;
if (memberCollective.type === types.USER && !memberCollective.isIncognito) {
  memberCollectiveUser = await models.User.findOne({ where: { CollectiveId: memberCollective.id } });
}
```

Comment (snitin315) [reply server/models/Collective.js]: Thanks

**Pull request merged**